src/components/LmsConfigurations/BlackboardIntegrationConfigForm.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import isEmpty from 'lodash/isEmpty';
 import {
-  ValidationFormGroup, Input, StatefulButton, Icon,
+  ValidationFormGroup, Input, StatefulButton, Icon, Hyperlink,
```

Comment:
Nit: Sometimes we like to put these on seperate lines to make future changes more readable